anko.go

```diff
@@ -17,75 +16,81 @@ import (
 	"github.com/mattn/anko/vm"
 
 	"github.com/daviddengcn/go-colortext"
-	"github.com/mattn/go-isatty"
 )
 
-const version = "0.0.2"
+const version = "0.0.3"
 
 var (
-	fs   = flag.NewFlagSet(os.Args[0], 1)
-	line = fs.String("e", "", "One line of program")
-	v    = fs.Bool("v", false, "Display version")
-
-	istty = isatty.IsTerminal(os.Stdout.Fd())
+	flagExecute *string
```

Comment:
please use non-pointer variable and flag.StringVar instead.

Reply:
Will do